azure/datalake/store/core.py

```diff
@@ -548,7 +548,8 @@ def concat(self, outfile, filelist, delete_source=False):
         self.azure.call('MSCONCAT', outfile.as_posix(),
                         data=bytearray(json.dumps(sources,separators=(',', ':')), encoding="utf-8"),
                         deleteSourceDirectory=delete,
-                        headers={'Content-Type': "application/json"},)
+                        headers={'Content-Type': "application/json"},
+                        retry_policy=ExponentialRetryPolicy(max_retries=0))
```

Comment:
It would have been better if we had a class like NoRetryPolicy. Also can you check what is the retry policy on create with overwrite=false, concurrentappend. That should be no retry also.

Reply:
Let me remove this change from here. It seems to be out of scope for this. I'll create a new pr for this.

Reply:
okay thanks.